docs/src/docs/browsercaps/common-browser-caps.adoc

```diff
@@ -23,19 +23,27 @@ WEB_DRIVER_MANAGER.setGlobalCapability(FirefoxOptions.FIREFOX_OPTIONS, options);
 
 ====
 
+= Request capabilities
 
-== Request capabilities
-
-Some WebDriverRequests support setting capabilities, like the `DesktopWebDriverRequest`.
+A request defined capability means using a Webdriver request like `DesktopWebDriverRequest`.
 
+.Set capabilities to a DesktopWebDriverRequest object
```

Comment:
Leading dot does not seem to do anything (and also does not get printed by Github).

Reply:
The leading dot means that the following text is formated as a description for the following code snippet.

Reply:
Mmh. Does not seem to have a tangible effect [when viewed in GitHub](https://github.com/telekom/testerra/blob/docs/add-links-to-tt-version-docs/docs/src/docs/browsercaps/common-browser-caps.adoc). Which platform is this supposed to be rendered correctly on?

Reply:
Please build it by your own with `gradle asciidoctor` and view the HTML result. You will find it under `docs/build/docs/html5/index.html`

Reply:
Ähm... yes, I forgot: #112

Reply:
Mmh. Works in IntelliJ though. Oh well.

Reply:
Had to adjust my `gradle-wrapper.properties` to use `gradle-6.9.2-bin.zip` (is 7.4), then it worked.

Reply:
Yeah! :-)